tests/scripts/components-configuration-crypto.sh

```diff
@@ -11,6 +11,28 @@
 
 CMAKE_BUILTIN_BUILD_DIR="drivers/builtin/CMakeFiles/builtin.dir/src"
 
+component_test_accel_ecdsa() {
+    msg "build: accelerated ECDSA"
+
+    # Build
+    # -----
+
+    cd $OUT_OF_SOURCE_DIR
+
+    cmake -DTF_PSA_CRYPTO_TEST_DRIVER=On \
+          -DTF_PSA_CRYPTO_USER_CONFIG_FILE="../tests/configs/user-config-accel-ecdsa.h" ..
```

Comment:
I appreciate the simplicity of having a ready-to-use file instead of a convoluted script. However, we are losing the ability to enumerate all the curves (and all the key pair sub-features). If we add a curve, it won't be enabled automatically unless we remember to add it to the config.

I'm not sure, is that ok? I think that if we forget a curve in the accel list, then the library will see that ECDSA can only be partially accelerated, and so `ecdsa.c.o` won't be empty, and we'll get a CI failure below. However, this is gray-box testing: we're assuming that the library implements this logic to enable built-in ECDSA if some curves are not accelerated. Does that matter? Are we validating that logic elsewhere?

Reply:
As explained below, in the next PR I reduce to one the number of user config file needed for all ECC acceleration components: `user-config-accel-ecc.h`. The file defines all the possible  `MBEDTLS_PSA_ACCEL_` ECC macros. Then we could have a check component checking that `user-config-accel-ecc.h` is up to date. Wdyt?